src/strings/csv/en.csv

```diff
@@ -1264,6 +1264,9 @@ OBSERVATIONS_EMPTY_STATE_MESSAGE_2,Observations with assigned plots can be sched
 OBSERVATIONS_EMPTY_STATE_TITLE,No Observations Yet,
 OBSERVATIONS_MAP_VIEW_PROMPT,"In order to access map view, please select a single planting site.",
 OBSERVATIONS_REQUIRED_BY_DATE,{0} monitoring plots require observation by {1}.,"Example: 5 monitoring plots require observation by June 30, 2023"
+OBSERVATIONS_TERRAWARE_MOBILE_APP_DESCRIPTION,"You will need the Terraware Mobile App to complete your observations.
+
+Scan the QR codes with your device to download on the App Store or Google Play.",
```

Comment:
It'd be great to include the French and Spanish translations too so they don't require a followup PR. You can keep the autotranslate tool running in the background and it'll update those CSV files continuously as you edit the English one.